packages/office-addin-test-helpers/test/test.ts

```diff
@@ -4,24 +4,30 @@
 import * as assert from "assert";
 import * as mocha from "mocha";
 import * as testHelper from "../src/testHelpers";
-import { TestServer } from "../../office-addin-test-server";
+import { defaultHttpPort, defaultHttpsPort, TestServer } from "../../office-addin-test-server";
 const port: number = 4201;
 const testServer = new TestServer(port);
-const promiseStartTestServer = testServer.startTestServer(true /* mochaTest */);
+const promiseStartTestServer = testServer.startTestServer();
 const testKey: string = "TestString";
 const testValue: string = "Office-Addin-Test-Infrastructure";
 const testValues: any = [];
 
 describe("Start test server, validate pingTestServer and sendTestResults methods and stop test server", function () {
     describe("Setup test server", function () {
         it("Test server should have started", async function () {
+            testHelper.setTLSRejectUnauthorized(false);
             const startTestServer = await promiseStartTestServer;
             assert.equal(startTestServer, true);
         });
         it("Test server should have responded to ping", async function () {
-            const testServerResponse: object = await testHelper.pingTestServer(port);
-            assert.equal(testServerResponse["status"], 200);
-            assert.equal(testServerResponse["platform"], testServer.getPlatformName());
+            // ping test server on https port
+            const httpsTestServerResponse: object = await testHelper.pingTestServer(defaultHttpsPort);
+            assert.equal(httpsTestServerResponse["status"], 200);
+            assert.equal(httpsTestServerResponse["platform"], testServer.getPlatformName());
+            // ping test server on http port
+            const httpTestServerResponse: object = await testHelper.pingTestServer(defaultHttpPort, false /* https */);
```

Comment:
The fact that you have `false ?* https */` here along with defaultHttpPort concerns me. Why have two different parameter to say whether or not to use https? I think it would be better to make pingTestServer just have a useHttps: boolean parameter, and it can know that it needs to use: `const port = useHttps ? defaults.httpsPort : defaults.httpPort;`

Reply:
made suggested change locally